src/app/static/src/app/components/adr/ADRInputDataset.vue

```diff
@@ -198,7 +197,11 @@ interface Data {
                     await this.deleteBaselineFiles()
                 }
 
-                await this.getDataset({id: this.newDatasetId!, release: this.newDatasetReleaseId});
+                await this.getDataset({
+                    id: this.newDatasetId!,
+                    release: this.newDatasetReleaseId,
+                    datasetType: AdrDatasetType.Input
```

Comment:
Oops! Missed the datasetType here in the last PR :see_no_evil:

Reply:
ahh is that why everything was breaking?